frontend/components/product-list/sections/FilterableProductsSection/useFacets.tsx

```diff
@@ -0,0 +1,9 @@
+import { useDynamicWidgets } from 'react-instantsearch-hooks-web';
+
+export function useFacets() {
+   const { attributesToRender } = useDynamicWidgets({
+      maxValuesPerFacet: 200,
```

Comment:
This seems unnecessarily high. Probably anything about 30-40 is on the edge of what's useful. Thoughts?

Reply:
I don't really have opinion about this, it depends on the max number of facets items. I've put this number because I think we were discussing with @sterlinghirsh that some facets might have a high number of items. Let me know what to set this to